src/puppetlabs/puppetdb/http/metrics/v1.clj

```diff
@@ -0,0 +1,16 @@
+(ns puppetlabs.puppetdb.http.metrics.v1
+  (:require [puppetlabs.puppetdb.http.metrics.core :as metrics]
+            [net.cgrand.moustache :as moustache]
+            [puppetlabs.puppetdb.middleware :refer
+             [wrap-with-debug-logging wrap-with-authorization
+              wrap-with-certificate-cn wrap-with-globals wrap-with-default-body]]
+            [ring.middleware.params :refer [wrap-params]]))
+
+(def v1-app
+  (moustache/app
+    ["mbeans"]
+    {:get metrics/list-mbeans}
+
+    ["mbean" & names]
+    {:get (moustache/app
+            (metrics/mbean names))}))
```

Comment:
Yeah we could also just use a case statement and put it in server.clj instead since its a pretty barebones app, i was just following the same convention we're using for the pdb app. Thoughts?